libs/wingsdk/src/target-tf-aws/counter.inflight.ts

```diff
@@ -35,4 +39,19 @@ export class CounterClient implements ICounterClient {
     // return the old value
     return parseInt(newValue) - amount;
   }
+
+  public async peek(): Promise<number> {
+    const command = new GetItemCommand({
+      TableName: this.tableName,
+      Key: { [HASH_KEY]: { S: COUNTER_ID } },
+    });
+
+    const result = await this.client.send(command);
+    let currentValue = result.Item?.[VALUE_ATTRIBUTE].N;
+    if (!currentValue) {
+      throw new Error(`${VALUE_ATTRIBUTE} attribute not found on table.`);
+    }
+
+    return parseInt(currentValue);
```

Comment:
Currently `Counter`'s implementation doesn't add an item to the DynamoDB table when the resource is deployed because doing so would require us to run code on deploy (something like [cdk triggers](https://docs.aws.amazon.com/cdk/api/v1/docs/triggers-readme.html)) - but such a resource/feature hasn't been added in the Wing SDK yet.

To work around this, we need to handle the situation where `inc` hasn't been called yet by returning the initial value if no item is found:
```suggestion
    const result = await this.client.send(command);
    let currentValue = result.Item?.[VALUE_ATTRIBUTE].N;
    if (!currentValue) {
      return this.initial;
    }

    return parseInt(currentValue);
```